Mathlib/Combinatorics/Additive/AP/Three/Defs.lean

```diff
@@ -429,11 +372,9 @@ theorem mulRothNumber_map_mul_left :
     obtain ⟨u, hus, rfl⟩ := hus
     rw [coe_map] at hu
     rw [← hcard, card_map]
-    exact (threeGPFree_mul_left_iff.1 hu).le_mulRothNumber hus
+    exact (threeGPFree_smul_set.1 hu).le_mulRothNumber hus
   · obtain ⟨u, hus, hcard, hu⟩ := mulRothNumber_spec s
-    have h : ThreeGPFree (u.map <| mulLeftEmbedding a : Set α) := by
-      rw [coe_map]
-      exact hu.mul_left
+    have h : ThreeGPFree (u.map <| mulLeftEmbedding a : Set α) := by rw [coe_map]; exact hu.smul_set
```

Comment:
This also needs more thinking than I'd like to identify mul and smul

Reply:
This is like that simply because I need something of the form `u.map something` to apply my lemmas